python-plugin/pom.xml

```diff
@@ -63,9 +61,25 @@
                     <pluginClass>fr.cnumr.python.CustomPythonRulesPlugin</pluginClass>
                     <sonarLintSupported>true</sonarLintSupported>
                     <sonarQubeMinVersion>${sonarqube.version}</sonarQubeMinVersion>
-                    <requirePlugins>python:${sonar.python.version}</requirePlugins>
+                    <!-- TODO : commented because plugin not loaded in SONAR if required version of python plugin is not installed -->
+                    <!-- TODO : check how to resolve it (how to force this version in SonarQube installation) -->
+                    <!-- <requirePlugins>python:${sonar.python.version}</requirePlugins> -->
+                    <basePlugin>python</basePlugin>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
due to the use of this maven plugin, some xml files appears (like `java-plugin/dependency-reduced-pom.xml` for example), please add this line to `.gitignore` file :
`dependency-reduced-pom.xml`

Reply:
done